.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: java
 
 jdk:
-  - oraclejdk8
+  - openjdk11
```

Comment:
Could you also add `oraclejdk11` as second build option. Would be great if we can test against openjdk and oraclejdk

Reply:
Done!